charts/telepresence/templates/deployment.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: apps/v1
```

Comment:
I'm sorry this wasn't obvious from the files we had in k8s/*.yaml and I should have caught it on the first review, but could we have these default to the `ambassador` namespace for all the things deployed to the cluster? Currently, the cli requires the service + deployment be in the ambassador namespace or else it will install another traffic-manager there.

Reply:
So setting the `metadata.namespace` would be very anti to the typical helm patterns.

Instead I just created a check that makes the install fail if you try to use any other namespace 🙂